tests/test_float16.py

```diff
@@ -21,7 +22,8 @@ def _ort_inference(mdl, inputs):
 Graph.opset = 9
 onnx_function = Graph.trace
 
-@unittest.skipIf(_ort.__version__ == '1.8.0', "see https://github.com/microsoft/onnxruntime/issues/7981")
+# @unittest.skipIf(StrictVersion(_ort.__version__) <= StrictVersion('1.8.0'), "see https://github.com/microsoft/onnxruntime/issues/7981")
```

Comment:
If this comment is not correct anymore, let's remove it directly.

Reply:
removed